app/src/write.js

```diff
@@ -31,13 +18,18 @@ module.exports = function(GulpAngularGenerator) {
    * Pass through each files and actually copy them
    */
   GulpAngularGenerator.prototype.writeFiles = function writeFiles() {
-    var process = processor(this);
+    this.props.angularModulesObject = {};
+    this.props.angularModules.map(function (module) {
+      this[module.key] = module.module;
+    }, this.props.angularModulesObject);
```

Comment:
@Swiip here I create a new `props` used in  `_bower.json` to replace `_.findWhere()` in template.